internal/config/config.go

```diff
@@ -282,6 +284,11 @@ func ParseDSN(dsn string) (UserConfig, error) {
 		ucfg.Location, err = time.LoadLocation(timezone)
 	}
 
+	if params.Has("stagingAllowedLocalPath") {
+		ucfg.StagingAllowedLocalPath = params.Get("stagingAllowedLocalPath")
+	}
+	params.Del("stagingAllowedLocalPath")
```

Comment:
If we parse this out of the DSN we should also allow the user to set it as a connector option.

Reply:
Now we've just moved it to the context.